ROADMAP.md

```diff
@@ -2,28 +2,20 @@
 
 This document defines a high level roadmap for Rook development. The dates below are subject to change but should give a general idea of what we are planning. We use the [milestone](https://github.com/rook/rook/milestones) feature in Github so look there for the most up-to-date and issue plan.
 
-## Rook 0.5 (July 2017)
-
- - Rook still in alpha
- - MON and OSD Reliability - survive restarts and pod failures
- - Backup of the cluster config and data required to recover it
- - Publish documentation to rook.io
- - Upgrade to Ceph Luminous and make the default
- - Long-haul testing infra
- - TPRs for object store and file system
- - Initial Rook Upgrade support
-
 ## Rook 0.6 (September 2017)
 
  - Ceph Block storage declared Beta on K8S
  - Kubernetes 1.8 Support
  - Rook volume plugin support in K8S
+ - Rook agent on client nodes
  - Rook can upgrade itself
- - Improved data placement and pool configuration
+ - Support for local storage (and arbitrary volumes) underneath OSD pods
  - Support for public cloud environments
- - Performance Testing infra
+ - Improved test matrix and more test coverage
+ - CRDs for object storage and file system
```

Comment:
Really the goal is that we are 100% kubernetes api and our own api service is just an optional component (for standalone or rookctl scenarios)